src/types/mpi.rs

```diff
@@ -157,7 +153,7 @@ impl<'a> Serialize for MpiRef<'a> {
 
 impl From<&[u8]> for Mpi {
```

Comment:
I wonder if we should remove this entirely and force direct calls instead

Reply:
also this is problematic, as this now behaves differently to `From<Vec<u8>>` as well as the `From` variants for `MpiRef`

Reply:
I think it's a good idea to remove both `From`s since it's not a simple conversion.

Reply:
Removing all `From` implementations sounds potentially right to me. We could try out how that would look, in this PR.

Reply:
Maybe *all* is a bit too much since there are many (e.g. for `BigUint`). The other approach would be having `from_slice` but inside stripping leading zeros (basically renaming `from_raw_slice` to `from_slice`).